instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -9,51 +9,34 @@ module Instrumentation
     module AwsSdk
       # Generates Spans for all interactions with AwsSdk
       class Handler < Seahorse::Client::Handler
-        def call(context)
-          span_name = get_span_name(context)
-          attributes = get_span_attributes(context)
+        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
These rubocop rules lead to some a scattering of really shallow methods, it's been a long standing "todo" to clean it up.